examples/analog_hamiltonian_simulation/04_Maximum_Independent_Sets_with_Analog_Hamiltonian_Simulation.ipynb

```diff
@@ -0,0 +1,569 @@
+{
```

Comment:
<p>The drawing algorithm used by networkx hides the unit disk property we wanted for this graph. Would it be better to just use the atom coordinates we already have to draw the graph?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/232/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>If there's an easy way to do this in networkx let's. </p>

Reply:
<p>Turns out there is.</p>